.github/workflows/test.yml

```diff
@@ -74,7 +74,7 @@ jobs:
         - name: mypyc runtime tests with py39-macos
           python: '3.9.18'
           arch: x64
-          os: macos-latest
+          os: macos-13
```

Comment:
Thanks! Could you add a comment here so we know to update it once we move the test to `3.10`?